wordpress-mobile/WordPress-iOS
# History / Revisions: Add swipe action to navigate trough revisions

Refs. #10304 

This PR introduces the swipe action into the Revisions browser.

![revisions-swipe](https://user-images.githubusercontent.com/912252/49367777-4cbdf680-f6ed-11e8-837b-9d8e11bc1a60.gif)

There's no a big change from the V1. A _UIPageViewController_ is used to handle the swipe and scroll. The addition of this class allowed me to keep the _RevisionDiffViewController_.
`RevisionDiffsPageManager` is a new class that handle the page view controller delegate and datasource and uses the `RevisionDiffsPageManagerDelegate` to comunicate with the revisions browser. This allowes me to keep the browser less verbose.

**To test:**
- Select a revision from a revisions list and open the revisions browser.
- You should be able to swipe between the revisions
- You should (still) be able to navigate between the revisions using the chevron

**Analytics:**
- The swipe gesture generate an analytics event: _revisions_detail_viewed_ with the param _swipe_
